src/requests/features.jl

```diff
@@ -190,22 +187,21 @@ function find_references(b::StaticLint.Binding, refs = EXPR[], from_end = false)
     end
 end
 
-JSONRPC.parse_params(::Type{Val{Symbol("textDocument/references")}}, params) = ReferenceParams(params)
-function process(r::JSONRPC.Request{Val{Symbol("textDocument/references")},ReferenceParams}, server)
-    return find_references(r.params.textDocument, r.params.position, server)
+function textDocument_references_request(conn, params::ReferenceParams, server)
+    # TODO The reference to `textDocument` here is a bug
```

Comment:
Not sure what the issue is here, why isn't it prefixed with `params.`?

Reply:
Ah, I had some mixup there, now fixed.